include/common/dm.h

```diff
@@ -22,7 +22,6 @@ enum {
 	DM_CLASS_MSGBOX,
 	DM_CLASS_PIO,
 	DM_CLASS_TIMER,
-	DM_CLASS_WALLCLOCK,
```

Comment:
why is this removed?

Reply:
Because it's no longer used (and won't be used in the future). It references removed code.